packages/imagemagick6.rb

```diff
@@ -55,6 +49,7 @@ def self.build
            ./configure \
            --prefix=#{CREW_PREFIX} \
            --libdir=#{CREW_LIB_PREFIX} \
+           --mandir=#{CREW_MAN_PREFIX} \
```

Comment:
Does `CREW_OPTIONS` work here?

Reply:
No. It works if you remove `--target` from `#{CREW_OPTIONS}`. Don't know why....